share/src/main/resources/alfresco/site-webscripts/org/alfresco/components/search/search.get_pt_BR.properties

```diff
@@ -15,8 +15,8 @@ label.document=Documento
 label.blogpost=Postagem de blog
 label.forumpost=T\u00f3pico do f\u00f3rum
 label.calendarevent=Evento do calend\u00e1rio
-label.wikipage=P\u00e1gina de wiki
-label.link=Link
+label.wikipage=P\u00e1gina Wiki
+label.link=Vincular
```

Comment:
I am not sure if you are using this for an action or what, but if the idea is to let the user add a link into the page, then the word Link is better, as we do not translate this word in Portuguese... Everybody uses the Link work to describe a page/website address